src/components/Hero/Hero.scss

```diff
@@ -0,0 +1,70 @@
+@import "~bootstrap/scss/functions";
+@import "~bootstrap/scss/variables";
+@import "~bootstrap/scss/mixins/_breakpoints";
+
+.hero {
+  position: relative;
+  overflow: hidden;
+
+  .container {
+    padding-left: 0;
+    padding-right: 0;
+
+    @include media-breakpoint-up(sm) {
+      & {
+        padding-left: 2rem;
+        padding-right: 2rem;
+      }
+    }
+  }
+
+  .background {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+
+    .hero-texture,
+    .hero-img {
+      background-color: #ffffff;
+      background-size: cover;
+      background-repeat: no-repeat;
+      background-position: center;
+    }
+
+    .hero-texture {
+      flex: 1;
+      flex-grow: 0;
+      flex-basis: 33.33%;
+      border-right-width: 20px;
+      border-right-style: solid;
+    }
+
+    .hero-img {
+      flex: 1;
+    }
+  }
+
+  .hero-box {
+    border-left-width: 20px;
+    border-left-style: solid;
+  }
+
+  .hero-logo-wrapper {
+    @include media-breakpoint-up(xs) {
+      & {
+        margin-top: 40px;
+      }
+    }
+
+    @include media-breakpoint-up(lg) {
+      & {
+        margin-top: 135px;
+      }
+    }
+
+    .hero-logo {
+      border-left-width: 20px;
+      border-left-style: solid;
+    }
+  }
+}
```

Comment:
@adamstankiewicz A css noob question, I want to understand why we are using breakpoints here? My understanding was that all the desired responsive classes are provided by bootstrap so we don't need to write custom css?